### DISCUSSION TITLE
Error: CDO Constructor: Failed to find /AirSim/Blueprints/BP_PIPCamera.BP_PIPCamera_C

### DISCUSSION THREAD

**Discussion by jueunkwon** [microsoft/AirSim#3616] [Support Q&A] [answered]
Hello. I was working with Sequence Recorder before and now, whenever I open LandscapeMountains.sln, I get an error that says "Failed to find /AirSim/Blueprints/BP_PIPCamera.BP_PIPCamera_C". Thank you.

![Error](https://user-images.githubusercontent.com/80252031/115434007-32114d00-a1d6-11eb-87cd-4d4db3e484bf.JPG)

**Comment by rajat2004** [comment, marked as answer]
I think you're using UE 4.24 currently? 4.25 is the currently supported version, I think before https://github.com/microsoft/AirSim/pull/3233, 4.24 worked with some small code change, however latest master won't. So you could either switch to an older commit of the master branch, or use 4.25. It should be straightforward to upgrade your environment, but best to have a copy as well for backup

**Reply by jueunkwon** [reply]
I am currently using UE 4.26 and didn't save a backup file--I will from now on :(
I am still not quite sure where this problem arose from. Thank you!

**Reply by jueunkwon** [reply]
I changed the version from UE 4.26 to UE 4.25 and the problem has been resolved. Thank you.
